beets/ui/commands.py

```diff
@@ -1450,7 +1450,7 @@ def modify_func(lib, opts, args):
 
 # move: Move/copy files to the library or a new base directory.
 
-def move_items(lib, dest, query, copy, album, pretend, confirm=False):
+def move_items(lib, dest, query, copy, album, pretend, confirm=False, export):
```

Comment:
This:

> ERROR: Failure: SyntaxError (non-default argument follows default argument (commands.py, line 1453))

Means that export must either come before confirm or have a default value.